manifests/telegraf/install.pp

```diff
@@ -4,7 +4,9 @@
 #
 # @api private
 class puppet_metrics_dashboard::telegraf::install {
-  package { 'telegraf':
-    ensure  => present,
+  unless $puppet_metrics_dashboard::configure_telegraf {
```

Comment:
This is a little odd... `puppet_metrics_dashboard::telegraf::config` installs the telegraf package through the telegraf module (there's no flag to disable that), so if we have `configure_telegraf => true` then that class will install the package (we'd get a duplicate resource declaration here with `if`)

In other words this class installs the package _unless_ we're configuring telegraf, in that case `puppet_metrics_dashboard::telegraf::config` installs the package.

Reply:
Ah that makes sense, thanks for the explanation. Should we just remove this package management and let the telegraf module handle it, since the only use case for installing the package is when we are configuring telegraf?